src/lib/helpers/yargs/options.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+
+const withToken = yargs => {
+	return yargs.option("token", {
+		describe:
+			"GitHub personal access token (uses `GITHUB_PERSONAL_ACCESS_TOKEN` environment variable by default). Generate one at https://github.com/settings/tokens.",
```

Comment:
**Problem**
This error is helpful, but does not say which permissions the GITHUB_PERSONAL_ACCESS_TOKEN requires.

**Solutions?**

- Do nothing
- Change the error message to include "GitHub personal access token with all `repo` scopes"
- Change the error message to point to more details in the README?
- Change the URL to preselect the required permissions? (E.g. https://github.com/settings/tokens/new?description=@financial-times/github&scopes=repo)

Reply:
It's worth mentioning that this _isn't_ an error message (it's the description that is displayed for this option when `yargs` outputs help), but you've raised a good point that there's no longer documentation in the `README` that mentions GitHub personal access tokens. It went away [here](https://github.com/Financial-Times/github/commit/28c88ac5aa2d13356dcbde806b994062a8978f3a?diff=split#diff-04c6e90faac2675aa89e2176d2eec7d8L46). I agree that we shouldn't be linking to private internal documentation that non-FT folk can't access (which it was), but we should have _something_ in the README about tokens. I'm going to raise an issue for this.

Reply:
Nice one